tests/test_hep_bdFFT.py

```diff
@@ -544,3 +544,33 @@ def test_figures2marc_handles_unicode():
     result = hep2marc.do(record)
 
     assert expected == result['FFT']
+
+
+def test_documents_from_FFT_without_t_subfield():
+    schema = load_schema('hep')
+    subschema = schema['properties']['documents']
+
+    snippet = (
+        '<datafield tag="FFT" ind1=" " ind2=" ">'
+        '  <subfield code="a">/opt/cds-invenio/var/data/files/g151/3037619/content.pdf;1</subfield>'
+        '  <subfield code="d"/>'
+        '  <subfield code="f">.pdf</subfield>'
+        '  <subfield code="n">arXiv:1710.01187</subfield>'
+        '  <subfield code="r"/>'
+        '  <subfield code="s">2017-10-04 09:42:00</subfield>'
+        '  <subfield code="v">1</subfield>'
+        '  <subfield code="z"/>'
+        '</datafield>'
+    )  # record/1628455
```

Comment:
is this real data? If I open that record the field has a `t`.

Reply:
No, the test case itself is based on the other test cases: `test_figures_and_documents_from_FFT_without_d_subfield`.

In SCOAP3 we receive xml, that doesn't contain this subfield from IOP.

Reply:
yeah, you shouldn't do that. We try to use real data if possible in the tests here, so we are sure we test real use cases. So you can put a real FFT you're seeing here.